functions.qmd

```diff
@@ -166,26 +166,27 @@ rescale01 <- function(x) {
   rng <- range(x, na.rm = TRUE, finite = TRUE)
   (x - rng[1]) / (rng[2] - rng[1])
 }
+
```

Comment:
Throughout I tried to format the code so that it looks the same, i.e. adding a space whenever a new function has been defined. Please ignore if it this is unnecessary. It's just something I noticed.